devtools/datasette/publish.py

```diff
@@ -27,7 +27,8 @@
 
 import click
 
-from pudl.helpers import check_tables_have_metadata, create_datasette_metadata_yaml
+from pudl.helpers import check_tables_have_metadata
+from pudl.metadata.classes import DatasetteMetadata
```

Comment:
@e-belfer Turns out that `create_datasette_metadata_yaml` function that was using this import was:
* only used in one place
* 3 lines long

So I decided to inline it in the one place it's used. One fewer circular import to deal with!

Reply:
There's a reference to it in `test/integration/datasette_metadata_test.py` (about how we're not using it there) which should get updated/removed.

Reply:
Ah, you mean in the comment? I can remove that :)